version/version.go

```diff
@@ -14,9 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package version
+package consts
 
 const (
+	// BundleVersionAnnotation is the annotation key used in the Gateway API inference extension CRDs to specify
+	// the installed Gateway API inference extension version.
+	BundleVersionAnnotation = "inference.networking.x-k8s.io/bundle-version"
```

Comment:
Since we expect this to be long lived, I'd recommend using k8s.io instead of x-k8s.io
```suggestion
	BundleVersionAnnotation = "inference.networking.k8s.io/bundle-version"
```